lib/feature.pm

```diff
@@ -388,51 +388,47 @@ The following feature bundles are available:
   --------- -----------------
   :default  indirect
 
-  :5.10     say state switch indirect
+  :5.10     indirect say state switch
```

Comment:
I realize this is only a POD change but it's a big re-sort. What is the new sorting algorithm?

Reply:
it's now using an alphabetical sort making it easier to read and parse